### PR TITLE
Update installing_heater_with_rtv.md

### PR DESCRIPTION
**Pull request by camerony**
I contacted Keenovo about the use of RTV for 3d printers and they responded that instructions for the use of RTV is for sealing oil pans.   I feel that the use of RTV for the reinforcement of the heater pad that already have an adhesive should not be recommended since it's an unnecessary safety hazard.

"
Johnny Tang | KEENOVO <sales@keenovo.com>
Wed, Mar 9, 4:02 AM (1 day ago)
to me

Hi,

Thank you for the contact!

Actually, the best approach from our perspective is a clamp as explained in this post:

https://keenovo.store/blogs/how-to/how-to-achieve-a-perfect-installation-of-a-keenovo-heater-pad-to-your-3d-printers-build-plate
Basically, the RTV sealing is mostly suggested for engine oil pan heaters.

Thank you!
Please feel free to contact us if you have any questions.
Best Regards,

Johnny Tang | Sales Department

Mobile:+86 186 1635 0189"

### CODE REVIEW

**Comment by bdbell** [comment]
Please contact the original author to get their approval on the change.

**Comment by camerony** [comment]
I actually tried via Discord 3/9/2022, and have not gotten a response yet. 
9964184F-2ECF-4A97-9EBB-56628FC458E2

**Comment by clee** [comment]
@Fulg any comments on this one?

**Comment by Fulg** [comment]
Hello, sorry I don't respond to most DMs on Discord! I think an important detail has been omitted here. RTV is supposed to _replace_ the adhesive, not be used _in addition to_. You have to special order your heater from Keenovo _without adhesive_ in order to use RTV on it. You cannot use both, otherwise that is indeed a safety hazard. Maybe this bit isn't clear in the text I wrote back then.

If you don't want to use RTV, I know Fabreeko sells heaters that use a more appropriate adhesive from Lohmann which fits the temperatures we use for printing ABS (as opposed to the 3M 468MP adhesive used by everyone in the market). I don't know if you can buy the Lohmann adhesive separately, or if it would be possible to convince Keenovo to use it for heaters destined to 3D printers.

**Comment by Fulg** [comment]
That being said I agree with the proposed changes, I think the details only serve to confuse readers. I would even put the disclaimer larger.

**Comment by camerony** [comment]
Added more emphasis.
